process.c

```diff
@@ -4679,56 +4679,132 @@ rb_spawn(int argc, const VALUE *argv)
+ *  it may inherit some of its environment from the calling program
+ *  (possibly including open file descriptors).
  *
- *     system("echo *")
- *     system("echo", "*")
+ *  Argument +env+, if given, is a hash that affects +ENV+ for the new process;
+ *  see {Execution Environment}[rdoc-ref:Process@Execution+Environment].
  *
- *  <em>produces:</em>
+ *  Argument +options+ is a hash of options for the new process;
+ *  see {Execution Options}[rdoc-ref:Process@Execution+Options].
+ *
+ *  The first required argument is one of the following:
+ *
+ *  - +command_line+ if it is a string,
+ *    and if it begins with a shell reserved word or special built-in,
+ *    or if it contains one or more metacharacters.
+ *  - +exe_path+ otherwise.
+ *
+ *  <b>Argument +command_line+</b>
+ *
+ *  \String argument +command_line+ is a command line to be passed to a shell;
+ *  it must begin with a shell reserved word, begin with a special built-in,
+ *  or contain meta characters:
+ *
+ *    system('echo')                             # => true  # Built-in.
+ *    system('if true; then echo "Foo"; fi')     # => true  # Shell reserved word.
+ *    system('date > /tmp/date.tmp')             # => true  # Contains meta character.
+ *    system('date > /nop/date.tmp')             # => false
+ *    system('date > date.tmp', exception: true) # Raises RuntimeError.
+ *
+ *  Assigns the command’s error status to global variable <tt>$?</tt>:
+ *
+ *    system('echo')                             # => true  # Built-in.
+ *    $?                                         # => #<Process::Status: pid 640610 exit 0>
+ *    system('date > /nop/date.tmp')             # => false
+ *    $?                                         # => #<Process::Status: pid 640742 exit 2>
+ *
+ *  The command line may also contain arguments and options for the command:
+ *
+ *    system('echo "Foo"') # => true
+ *
+ *  Output:
+ *
+ *    Foo
+ *
+ *  On a Unix-like system, the shell is <tt>/bin/sh</tt>;
+ *  otherwise the shell is determined by environment variable
+ *  <tt>ENV['RUBYSHELL']</tt>, if defined, or <tt>ENV['COMSPEC']</tt> otherwise.
  *
- *     config.h main.rb
- *     *
+ *  Except for the +COMSPEC+ case,
+ *  the entire string +command_line+ is passed as an argument
+ *  to {shell option -c}[https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/sh.html].
+ *
+ *  The shell performs normal shell expansion on the command line:
+ *
+ *    system('echo C*') # => true
+ *
+ *  Output:
+ *
+ *    CONTRIBUTING.md COPYING COPYING.ja
+ *
+ *  Raises an exception if the new process fails to execute.
+ *
+ *  <b>Argument +exe_path+</b>
+ *
+ *  Argument +exe_path+ is one of the following:
+ *
+ *  - The string path to an executable to be called.
+ *  - A 2-element array containing the path to an executable
+ *    and the string to be used as the name of the executing process.
+ *
+ *  Example:
+ *
+ *    system('/usr/bin/date') # => true
```

Comment:
Note that the location of `date` is system dependent.

Reply:
Noted.